skorch/tests/callbacks/test_logging.py

```diff
@@ -43,12 +43,7 @@ def neptune_logger_cls(self):
 
     @pytest.fixture
     def neptune_run_object(self):
-        try:
-            # neptune-client=0.9.0+ package structure
-            import neptune.new as neptune
-        except ImportError:
-            # neptune-client>=1.0.0 package structure
-            import neptune
+        import neptune
```

Comment:
We update `requirements.txt` to install latest `neptune` so no need to guard the imports for older version.